shared/routes/Repl/Viewer.html

```diff
@@ -17,10 +19,27 @@
 </div>
 
 <style>
+	.iframe-container {
+		border-top: 1px solid #ccc;
+	}
+
 	iframe {
 		width: 100%;
-		height: 100%;
+		height: calc(100vh - 3em);
 		border: none;
+		background-color: white;
```

Comment:
I can show you screenshots of bad cases, when I'm working on laptop (maybe at weekend).